packages/messenger-widget/src/interfaces/context.ts

```diff
@@ -38,3 +38,12 @@ export interface UiState {
 export interface GlobalContextProviderProps {
     children: JSX.Element;
 }
+
+export interface RightView {
```

Comment:
> Could this be done with an Enumeration? e.g.
> 
> ```
> enum SelectedRightView {
>   Header,
>   Profile,
>   ...
> }
> ```
> 
> Because it shouldn't be possible to show the profile and the chat at the same time

Yes, it can be done.

Reply:
I will change this in the next PR, is it fine?

Reply:
Yes next PR is fine